src/godot.ts

```diff
@@ -151,7 +151,7 @@ async function doExport(): Promise<BuildResult[]> {
 
     let executablePath;
     if (preset.export_path) {
-      executablePath = path.join(buildDir, path.basename(preset.export_path));
+      executablePath = path.join(RELATIVE_PROJECT_PATH, preset.export_path);
```

Comment:
I'm afraid this isn't going to work as cleanly as you might hope. Let's say a user has their export path defined simply as `windows.exe`. What this will do, is build the `windows.exe` and `game.pck` at the same level as `project.godot`. Then, when a release is created it will zip up ALL of the contents of the folder, including all of the raw development resources. This was the reason for building the games to a separate builds directory, so that the entirety of the directory could be safely zipped in its entirety.

You could approach this in a couple of ways:
- You can check to make sure the directories the builds are going to are empty (you'd also have to make sure that multiple presets aren't all pointing to the same directory)
- You can keep the builds going to a separate directory and modify [this function](https://github.com/firebelley/godot-export/blob/71657a8f8e3b4021f531ae9362d9966dfb47d52c/src/file.ts#L37) to behave slightly differently

Alternatively we could just deal with the potential for user error, but I fear that this functionality won't be totally clear and will result in some undesirable effects. Also, I'd like to see some way to control this behavior via an optional input to the action.

Looking forward to hearing back from you! Thanks again for the contribution.

Reply:
Ah yes, I see my issue. I think I have an idea though, we could by default set `relative_export_path` to be the export_path. So in my case where I have `export_path="HTML5/index.html"`, if I leave `relative_export_path` empty it would be equal to `HTML5/`.
I wanna know what you think of this idea.

Reply:
Yes this sounds like a good approach, I think the default behavior should still be to keep builds in that transient `builds` directory unless the user specifies that they want the builds exported in accordance with the paths in `export_presets.cfg`. Keeping the builds in that `builds` directory just makes a lot of operations easier.

Maybe a new optional input, something like `move_exports` or `relative_exports` that when set to `true` will use the `export_presets.cfg` paths relative to the project root. That makes sense to me. I'm not super concerned with the details, I just think it would be nice to control that behavior. I personally don't have a use-case currently where I want my exports to exist anywhere in particular, I just want builds for my release.

Something else to consider is there is an input where you can archive all the outputs: `archive_export_output`, we'll want to make sure this is adhered to still when using the path present in `export_presets.cfg`.

Just some context for you to consider, happy to talk about this further but I think overall your suggestion is a good one!